packages/react-ui/src/layout/Section/index.js

```diff
@@ -1,34 +1,84 @@
+import React from "react";
 import PropTypes from "prop-types";
 import styled, { css } from "styled-components";
-import { spacings } from "../../theme";
+import { breakpoints, spacings } from "../../theme";
 
-export const Section = styled.div`
-  padding: ${spacings.medium} 0;
+const assignBackgroundColor = ({ variant, theme }) => {
+  let backgroundColor = "transparent";
+  if (variant === "white") {
+    backgroundColor = theme.white;
+  }
+  if (variant === "light") {
+    backgroundColor = theme.bgSecondary;
+  }
+  if (variant === "dark") {
+    backgroundColor = theme.bgTertiary;
+  }
+  return css`
+    background-color: ${backgroundColor};
+  `;
+};
+
+export const Section = ({ children, decorated, variant, ...props }) => {
+  if (decorated) {
+    return (
+      <StyledSection decorated>
+        <Decoration variant={variant} />
+        <Content {...props}>{children}</Content>
+      </StyledSection>
+    );
+  }
+  return (
+    <StyledSection variant={variant} {...props}>
+      {children}
+    </StyledSection>
+  );
+};
+
+const StyledSection = styled.div`
+  margin: ${({ decorated }) => (decorated ? spacings.medium : 0)} 0;
```

Comment:
we should probablyd add a `min-height` when StyledSection is decorated to avoid weird border-radius effect ?

Reply:
Allez !